pkg/distributors/contracts/MerkleOrchard.sol

```diff
@@ -76,17 +78,144 @@ contract MerkleOrchard {
         IERC20 token,
         address distributor,
         uint256 distribution,
-        address liquidityProvider
+        address claimer
```

Comment:
Note that some comments list 'lp', for example on line 35 when describing the mapping layout.

Reply:
Thanks, I can't find any references other than this.